examples/wearable/UIComponents/contents/navigationelements/titlebar/title-subtitle-on-list-2lines.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta content="width=device-width, initial-scale=1, user-scalable=no" name="viewport" />
+	<title>
+		Title with subtitle on list with 2 subitems
+	</title>
+	<link href="../../../lib/tau/wearable/theme/default/tau.css" rel="stylesheet" />
+	<link href="../../../lib/tau/wearable/theme/default/tau.circle.css" rel="stylesheet" />
+	<link href="../../../css/style.css" rel="stylesheet" />
+	<link href="../../../css/style.circle.css" rel="stylesheet" />
+	<script src="../../../js/circle-helper.js">
+	</script>
+	<script src="../../../lib/tau/wearable/js/tau.js">
+	</script>
+</head>
+
+<body>
+	<div class="ui-page">
+		<header class="ui-header-on-list-with-2-subitems">
```

Comment:
I'm wondering about this class name.
Seems too long. Maybe we can use name convention like for list items, eg. `title-has-2line-sub`

Reply:
has-2line-sub can suggest 2 subtitles in title,
I was trying different abbreviation but seems no good way to shorten it

Reply:
As we discussed let use header medium instead of new class.

Reply:
done